core-web/libs/data-access/src/lib/dot-events/dot-events.service.ts

```diff
@@ -24,7 +24,7 @@ export class DotEventsService {
         // TODO: need to make this method to support multiple events
         return this.subject
             .asObservable()
-            .pipe(filter((res: DotEvent<T>) => res.name === eventName));
+            .pipe(filter((res) => res.name === eventName)) as Observable<DotEvent<T>>;
```

Comment:
why `as`? are we typing this correctly?

Reply:
Typescript error. I didn't dig into it, not part of the PR but it was throwing a warning